discovery-provider/src/queries/notifications.py

```diff
@@ -1182,7 +1197,13 @@ def solana_notifications():
                     const.solana_notification_metadata: {
                         const.solana_notification_reaction_type: reaction.reaction_type,
                         const.solana_notification_reaction_reaction_value: reaction.reaction_value,
-                        const.solana_notification_reaction_reacted_to: reaction.reacted_to,
+                        const.solana_notification_reaction_reacted_to_entity: {
```

Comment:
hmm it's sad we can't serialize the tip into its model here

Reply:
You mean the flask model? yeah - although we don't really need every field here for the tip